pkg/virt-handler/cache/cache.go

```diff
@@ -299,10 +301,18 @@ func (d *DomainWatcher) startBackground() error {
 	go func() {
 		defer d.wg.Done()
 
+		resyncTicker := time.NewTicker(d.resyncPeriod)
+		resyncTickerChan := resyncTicker.C
+		defer resyncTicker.Stop()
+
 		// Divide the watchdogTimeout by 3 for our ticker.
 		// This ensures we always have at least 2 response failures
 		// in a row before we mark the socket as unavailable (which results in shutdown of VMI)
-		expiredWatchdogTicker := time.NewTicker(time.Duration((d.watchdogTimeout/3)+1) * time.Second).C
+		expiredWatchdogTicker := time.NewTicker(time.Duration((d.watchdogTimeout/3)+1) * time.Second)
+		defer expiredWatchdogTicker.Stop()
```

Comment:
Is this change related to the PR? or you just closed a leak that we had? It's it's the latter, could you mention it in the PR description?

Reply:
done